core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -112,6 +121,9 @@ export class NavigationComponent implements OnInit {
     this.runDisable = initBehavior.disable;
     this.onClickRunHandler = initBehavior.onClick;
     // this.currentWorkflowName = this.workflowCacheService.getCachedWorkflow();
+    if (this.router.getCurrentNavigation()?.extras.state?.execution) {
```

Comment:
no need for this, we can just get the execution info from `executionservice`

Reply:
refactored